website/content/docs/what-is-vault.mdx

```diff
@@ -51,16 +65,75 @@ The key features of Vault are:
   Revocation assists in key rolling as well as locking down systems in the
   case of an intrusion.
 
-## What is HCP Vault?
+-> **Tip:** Learn more about [Vault use cases](/docs/use-cases).
```

Comment:
FWIW, Laura and I have been using the block quote component instead of callout boxes on What is . . .? pages. This might be a personal thing, but I would also highlight "Use cases" as the keyword instead of "Tip" (or "Note", etc.) because I think it's more meaningful to readers when they are scanning.

Reply:
Thanks Adam. I'm going to have Laura weigh on your suggestion about highlighting 'use cases' instead of 'Tip'. I thought that was the convention used throughout our docs (and do see that convention used as well in Terraform docs). I tend to lean toward highlighting the 'Tip' or 'Note' since linking to the 'use cases' automatically highlights the words.